### PR TITLE
feat(arm): ARM FunctionAppHttpVersionLatest policy

### PR DESCRIPTION
**Pull request by RachelBorzi**
**By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license.**

## Description

Added new policies for -
FunctionAppHttpVersionLatest - CKV_AZURE_67 
and AppServiceDisallowCORS - CKV_AZURE_57

### Description
I added a test that checks whether a value with a value of true is found under a certain field

### Fix
*How does someone fix the issue in code and/or in runtime?*

## Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] I have added tests that prove my feature, policy, or fix is effective and works
- [x] New and existing tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

File: checkov/arm/checks/resource/FunctionAppHttpVersionLatest.py [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+from checkov.common.models.enums import CheckCategories
+
+
+class FunctionAppHttpVersionLatest(BaseResourceValueCheck):
+
+    def __init__(self):
+        name = "Ensure that 'HTTP Version' is the latest, if used to run the Function app"
+        id = "CKV_AZURE_67"
+        supported_resources = ("Microsoft.Web/sites/slots",)
+        categories = [CheckCategories.GENERAL_SECURITY]
```
Comment (ChanochShayner) [inline checkov/arm/checks/resource/FunctionAppHttpVersionLatest.py:11]: Please use a tuple instead of a list.

**Review by ChanochShayner — COMMENTED** [review]
Great Job!
Please update the PR description with the relevent info, and fix the failing jobs.

**Review by ChanochShayner — APPROVED** [review]
💯

**Review by lirshindalman — APPROVED** [review]

File: checkov/arm/checks/resource/FunctionAppHttpVersionLatest.py [hunk 2]
```diff
@@ -0,0 +1,22 @@
+from checkov.arm.base_resource_value_check import BaseResourceValueCheck
+from checkov.common.models.enums import CheckCategories
+
+
+class FunctionAppHttpVersionLatest(BaseResourceValueCheck):
+
+    def __init__(self) -> None:
+        name = "Ensure that 'HTTP Version' is the latest, if used to run the Function app"
+        id = "CKV_AZURE_67"
+        supported_resources = ("Microsoft.Web/sites/slots",)
```
Comment (tsmithv11) [inline checkov/arm/checks/resource/FunctionAppHttpVersionLatest.py:10]:
```suggestion
        supported_resources = ("Microsoft.Web/sites/slots", "Microsoft.Web/sites")
```

Sites can also have this set: https://learn.microsoft.com/en-us/azure/templates/microsoft.web/sites?pivots=deployment-language-arm-template#siteconfig-1

Can you add a UT for this?

**Review by tsmithv11 — APPROVED** [review]
Looks good! One suggestion

**Comment by RachelBorzi** [comment]
I'll take care of that, thanks.

‫בתאריך יום ד׳, 8 במאי 2024 ב-12:07 מאת ‪Taylor‬‏ ***@***.***
‬‏>:‬

**Pull request merged**